src/base/static/libs/maps/index.js

```diff
@@ -0,0 +1,533 @@
+import MapboxGLProvider from "./mapboxgl-provider";
+// Import other providers here as they become available
+
+var Util = require("../../js/utils.js");
+var LayerView = require("mapseed-layer-view");
+
+class MainMap {
+  constructor({ container, places, router, store }) {
+    this.mapConfig = store.getState().config.map;
```

Comment:
Is this the correct use of the config duck in this context?

Reply:
I suggest passing `mapConfig` directly from app-view, by importing `mapConfigSelector` into app view, and passing `mapConfigSelector(store.getState())` into the MainMap `mapConfig` constructor arg.

For now, MainMap should be decoupled from Redux specifics.